microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -0,0 +1,163 @@
+
+You can set browser flags in your local device environment, or set browser flags programmatically through code.  Both approaches are described below.
+
+
+<!-- ====================================================================== -->
+## Setting browser flags in your local device environment
+
+For testing forthcoming features or diagnosing issues, we recommend using browser flags in your local device environment, via setting the `WEBVIEW2_BROWSER_ARGUMENTS` environment variable or via registry keys.  For more information, see `CreateCoreWebView2EnvironmentWithOptions`:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CreateCoreWebView2EnvironmentWithOptions]<!-- todo: url for .NET equivalent of the Win32 global `CreateCoreWebView2EnvironmentWithOptions`? -->
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CreateCoreWebView2EnvironmentWithOptions]<!-- todo: url for WinRT equivalent of the Win32 global `CreateCoreWebView2EnvironmentWithOptions`? -->
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [CreateCoreWebView2EnvironmentWithOptions](/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions)
+
+---
+
+
+<!-- ====================================================================== -->
+## Setting browser flags programmatically through code
+
+Or, you can set browser flags programmatically, by passing the browser flags as the `AdditionalBrowserArguments` property of `EnvironmentOptions`:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2EnvironmentOptions` Class:
+   * [CoreWebView2EnvironmentOptions.AdditionalBrowserArguments Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.additionalbrowserarguments)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2EnvironmentOptions` Class:
+   * [CoreWebView2EnvironmentOptions.AdditionalBrowserArguments Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#additionalbrowserarguments)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* `ICoreWebView2EnvironmentOptions`:
+   * [ICoreWebView2EnvironmentOptions::get_AdditionalBrowserArguments](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions#get_additionalbrowserarguments)
+   * [ICoreWebView2EnvironmentOptions::put_AdditionalBrowserArguments](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions#put_additionalbrowserarguments)
+
+---
+
+If you set browser flags programmatically, you should remove the flags in code before shipping your app, to avoid accidentally shipping the flags in production.
+
+
+<!-- ====================================================================== -->
+## Available WebView2 browser flags
+
+The following are some of the flags we've seen used.<!--todo: explain why we're (reactively) reporting some of the flags ppl have used, rather than reporting the complete list of all flags that are supported, from eg a Chromium spec. -->
```

Comment:
state why we're (reactively) reporting some of the flags ppl have used, rather than reporting the complete list of all flags that are supported, from eg a Chromium spec
@nishitha-burman

Reply:
kept; removed hidden comment